src/playground/sample_data_tooling/sample_data_generator/USAGE.md

```diff
@@ -0,0 +1,31 @@
+## Example: Police ticketing data
```

Comment:
should we use the AD as sample? which they are asking

Reply:
Yes I included an example down below for AD team